docs/gathered-data.md

```diff
@@ -979,6 +964,23 @@ gathers the alerts that have been silenced.
   - 4.10+
 
 
+## StorageCluster
```

Comment:
probably no

Reply:
Good question. I think we should.

Reason: Rules should make conclusions based on data that are present in an archive (as opposed to data that are NOT present in the archive). If there are no data (for a particular resource), it can mean any of

1. No such resources in the cluster
2. Not gathered in the cluster version
3. An error occurred during gathering

To tell the three cases apart, rules use the gathers info. They need to know the gatherer name for that.

Reply:
Correction: My previous comment applies to config ID rather than gather name itself. We probably don't need to document the original gatherer name, but we should document the original config ID.

Reply:
So it's this discussion right https://github.com/openshift/insights-operator/pull/691#discussion_r1004214654

Reply:
yep